pom.xml

```diff
@@ -26,6 +26,9 @@
         <maven-pmd-plugin.version>3.15.0</maven-pmd-plugin.version>
         <maven-checkstyle-plugin.version>3.2.0</maven-checkstyle-plugin.version>
         <checkstyle.version>10.3.2</checkstyle.version>
+        <junit-jupiter-api.version>5.8.2</junit-jupiter-api.version>
```

Comment:
IDEA is highlighting these versions as not the latest. Please check whether we can update them to latest one